supervisor/supervisor.py

```diff
@@ -57,6 +62,14 @@ def __init__(self, coresys: CoreSys):
         self.coresys: CoreSys = coresys
         self.instance: DockerSupervisor = DockerSupervisor(coresys)
         self._connectivity: bool = True
+        self._connectivity_client: aiohttp.ClientSession = (
+            self._create_connectivity_client()
+        )
+
+    def _create_connectivity_client(self) -> aiohttp.ClientSession:
+        """Create a new connectivity client."""
+        connector = aiohttp.TCPConnector(use_dns_cache=False, force_close=True)
```

Comment:
The idea behind force_close is to avoid that the connectivity check reuses an existing connection. Right now, force_close wouldn't be necessary since default keep-alive is 15s, and our connectivity check is scheduled no more often than every 30s.

I was hoping that `use_dns_cache` avoids the necessity to recreate the session. But it seems without recreating the session `aiohttp` continous to raise `TimeoutError` even after the DNS issue has been cleared (as described in #5332).